Dungeon_of_Munchkinomicon.py

```diff
@@ -20,19 +20,28 @@ def death(name, reason):
 	print "You died because %s. What did you think would happen?" % reason
 	redo(name)
 	
+def getConfirmation(ask):
+        print ask
+        choice = raw_input(prompt).lower()
+        print choice
+        if "yes" in choice :
+                return True
+        else:
+                return False
+
+
 def redo(name):
 	global fairy
 	global katana
-	print "Try again, %s?" % name
-	tryagain = raw_input(prompt)
-	if "yes" in tryagain or "Yes" in tryagain:
+	tryAgain = getConfirmation("Try again, %s?" % name)
+	if tryAgain:
 		if name == "Cthulu":
-			username
-		else:
-			fairy = False
-			katana = False
-			
-			entrance(name)
+			name = username

```

Comment:
Would also need to change name to the new username to go with the above comment. 

Also, I don't know python standards but it seems logical that you may want to say username(). Both should work (?) the same but by adding the parens it is easy for new contributors looking at this to know that name is getting set to the results of a function and not just a variable username.